src/modules/alwaysontop/AlwaysOnTopModuleInterface/dllmain.cpp

```diff
@@ -72,15 +72,9 @@ class AlwaysOnTopModuleInterface : public PowertoyModuleIface
 
     // Return JSON with the configuration options.
     // These are the settings shown on the settings page along with their current values.
-    virtual bool get_config(wchar_t* buffer, int* buffer_size) override
+    virtual bool get_config(wchar_t* /*buffer*/, int* /*buffer_size*/) override
     {
         return false;
-        HINSTANCE hinstance = reinterpret_cast<HINSTANCE>(&__ImageBase);
-
-        // Create a Settings object.
-        PowerToysSettings::Settings settings(hinstance, get_name());
-
-        return settings.serialize_to_buffer(buffer, buffer_size);
```

Comment:
@SeraphimaZykova , Why does this have a return false on top? Shouldn't this code be called instead of deleted?

Reply:
You're right, seems like the return on top was forgotten here:  https://github.com/microsoft/PowerToys/pull/18643.
We need to keep this code and remove early return.

Reply:
Looking to the dates of the changes it makes sense to remove the "return false", although it was not what was running there

Changed